src/test/resources/pages/InstructorCourseDetailsWithSections.html

```diff
@@ -264,7 +264,7 @@
          <tbody>
             <tr class="student_row" id="student-c0.0">
                <td id="studentphoto-c0.0">
-                  <div class="profile-pic-icon-click align-center" data-link="/page/studentProfilePic?studentemail={*}&courseid={*}&user=CCDetailsUiT.instr2">
+                  <div class="profile-pic-icon-click align-center" data-link="/page/studentProfilePic?studentemail={***}&courseid={***}&user=CCDetailsUiT.instr2">
```

Comment:
The biggest sin of all: student profile picture link. Repeated hundreds of times. This means the regex replace for that in God mode does _nothing_ all these while.